### PR TITLE
Update developer.md with Eclipse and Postgres info

### PR DESCRIPTION
**Pull request by nealmcb**
and use more common term "issue tracking" (in my experience).

### CODE REVIEW

File: docs/25_developer.md [hunk 1]
```diff
@@ -61,18 +59,19 @@ Developer Tools
...
+with a pre-configured [Eclipse](https://eclipse.org/) install,
+a bootstrapping Eclipse workspace,
 and a VM image which can be loaded into VirtualBox or other comparable
 virtualization platform. We provide these resources in order to both
 decrease new developers' ramp-up time as well as to standardize on
 specific versions of tools for development.
 
-We expect to provide the initial Eclipse snapshot and workspace in the
-week of 9 July 2017.
+Instructions for installing Eclipse and the plugins we leverage
```
Comment (dmzimmerman) [inline docs/25_developer.md]: s/leverage/use

File: docs/25_developer.md [hunk 2]
```diff
@@ -61,18 +59,19 @@ Developer Tools
...
+are at [README-ECLIPSE.md](../server/README-ECLIPSE.md).
 
 * [GitHub](https://github.com/FreeAndFair/ColoradoRLA) for distributed
-  version control, change tracking, and development documentation
+  version control, issue tracking, and development documentation
```
Comment (dmzimmerman) [inline docs/25_developer.md]: Issues and changes are not the same thing, but I think change tracking is inherent in version control, so it all works out.

Comment (nealmcb) [inline docs/25_developer.md]: Exactly - that was my sense.

File: docs/25_developer.md (hunk 1 above)
Comment (nealmcb) [inline docs/25_developer.md]: Done, in the 2nd commit.

**Comment by dmzimmerman** [comment]
If you rebase this branch I can merge it in.

**Pull request merged**